packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -506,7 +504,6 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     }
 
     if (
-      this.currentActivePosition &&
```

Comment:
should this maybe just check the type? Something like...

- `this.currentActivePosition != null`?
- `typeof this.currentActivePostion === "number"`?

Reply:
Good catch, this made me realize we need that condition to support when all the item's are disabled. Will get that added.

Reply:
cool. yeah im guessing the issue was that the currentActivePostion was `0` which is a valid index number but causing this if statement to not get entered.

Reply:
yup.